actions/bump-and-release-cloud-plugin/README.md

```diff
@@ -0,0 +1,70 @@
+| `default-branch`            | The default branch of the repository                                                                                 | `false`  | `main`       |
+| `gcs-bucket`                | The GCS bucket to upload the artifacts to                                                                            | `true`   |              |
+| `gcs-service-account-creds` | The GCS service account credentials json to log into GCP                                                             | `true`   |              |
+| `github-token`              | The GitHub token to use for the action, requires the permissions to commit to main and create tags and releases.     | `true`   |              |
+| `go-version`                | The version of Go to use for building the plugin                                                                     | `false`  | `1.21`       |
+| `package-manager`           | The package manager used for the plugin                                                                              | `false`  | `yarn`       |
+| `release-level`             | The level of the release                                                                                             | `false`  | `prerelease` |
+| `signing-token`             | The access policy token to use for signing the plugin                                                                | `true`   |              |
+| `update-changelog`          | Whether to update the changelog file, set to true if you have your changelog automated with a `## Unreleased` header | `false`  | `false`      |
+| `validator-config-path`     | The path in the repo to the plugin-validator config file                                                             | `false`  |              |
+
+## Note on the artifacts
+
+The zip file name will be in the form `{{ plugin-id }}-{{ version }}.zip` E.g. `grafana-example-app-1.0.0.zip`
+
+There will also be a `*.zip.md5` checksum calculated and uploaded to the same GCS bucket.
+
+If the release level is anything other than `prerelease` an additional zip file will be uploaded as `*-latest.zip` (and
+checksum)
+
+## Example workflow
+
+```yaml
+name: Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      release-level:
+        description: 'Release level'
+        required: true
+        default: 'prerelease'
+        type: choice
+        options:
+          - major
+          - minor
+          - patch
+          - prerelease
+
+jobs:
+  bump-and-release:
+    runs-on: ubuntu-latest
+    steps:
+      # ... Steps to get necessary secrets
+      - uses: grafana/shared-workflows/actions/bump-and-release-cloud-plugin@main
+        with:
+          github-token: ${{ env.mygithub-token }} # Needs permission to commit to main and create tags and releases
+          gcs-bucket: ${{ env.GCS_BUCKET }}
+          gcs-service-account-creds: ${{ env.GCS_SERVICE_ACCOUNT_CREDS }}
```

Comment:
Making a comment here so we don't forget to remove this after OIDC is in place!

Reply:
hey @abannachGrafana 👋 
based on some recent discussion we believe that this PR should fit better in `grafana/grafana-github-actions` repo since it’s Grafana specific. we haven’t communicated this to people yet but this is where we are heading. wdyt?

Reply:
I'm fine with putting it there. I was more wondering about the OIDC portion that you were working on in deployment tools and what if any steps I need to take to make that happen in the action.